### DISCUSSION TITLE
Questions: Creating Local Trivy-Db

### DISCUSSION THREAD

**Discussion by BrittanyBoles7** [aquasecurity/trivy#5321] [Q&A]
### Question

Hello, I am struggling to understand the documentation a bit. I would like to given some specific local version of a vulnerability database, (NVD, Github Advisory etc.) convert it to a "trivy.db" and have Trivy scan only using the database I set. I am struggling to figure out how to get a database into the format necessary for Trivy to use. Any clarification in this regard would be helpful. 

### Target

None

### Scanner

Vulnerability

### Output Format

JSON

### Mode

None

### Operating System

ubuntu

### Version

```bash
all versions
```

**Comment by knqyf263** [comment]
There is no document about trivy-db internal. You can read the source code.
https://github.com/aquasecurity/trivy-db
